src/Toolkit/I18n.php

```diff
@@ -130,14 +110,12 @@ public static function locale(): string
 	/**
 	 * Translates a given message
 	 * according to the currently set locale
-	 *
-	 * @param string|array $key
-	 * @param string|array|null $fallback
-	 * @param string|null $locale
-	 * @return string|array|null
 	 */
-	public static function translate($key, $fallback = null, string $locale = null)
-	{
+	public static function translate(
+		string|array|null $key,
+		string|array|null $fallback = null,
+		string|null $locale = null
+	): mixed {
```

Comment:
Is it possible to return a value other than `string|array|null` types?

Reply:
Yes, e.g. in `I18nTest.php:360` it would be a Closure.